pulp_ansible/tests/functional/constants.py

```diff
@@ -38,15 +38,17 @@
 # FIXME: replace this with the location of one specific content unit of your choosing
 ANSIBLE_URL = urljoin(ANSIBLE_FIXTURE_URL, "")
 
-ANSIBLE_GALAXY_COLLECTION_URL = "https://galaxy-dev.ansible.com"
+ANSIBLE_GALAXY_COLLECTION_URL = "https://galaxy.ansible.com/api/v2/collections/"
+
+ANSIBLE_COLLECTION_FIXTURE_URL = urljoin(ANSIBLE_GALAXY_COLLECTION_URL, NAMESPACE_ANSIBLE)
 
 ANSIBLE_COLLECTION_REMOTE_PATH = urljoin(BASE_REMOTE_PATH, "ansible/collection/")
 
 COLLECTION_WHITELIST = "testing.ansible_testing_content"
 
 ANSIBLE_COLLECTION_CONTENT_NAME = "ansible.collection_version"
 
-ANSIBLE_COLLECTION_FIXTURE_COUNT = 1
+ANSIBLE_COLLECTION_FIXTURE_COUNT = 114
```

Comment:
apparently, filters are not working for https://galaxy.ansible.com/api/v2/collections/
so this number could vary

Reply:
One idea I've seen before in pup_ansible tests when the live galaxy could change is to assert that there are "at least" that many units. That is already in place for the roles tests AIUI.